libs/server-sent-events/src/client.cpp

```diff
@@ -263,8 +293,22 @@ class FoxyClient : public Client,
     void on_read_body(boost::system::error_code ec, std::size_t amount) {
         boost::ignore_unused(amount);
         if (ec == boost::asio::error::operation_aborted) {
+            logger_("read HTTP response body aborted");
             return;
         }
```

Comment:
Here's the key: if the parser indicates that the chunked encoding is finished, initiate backoff instead of attempting to keep reading the body.